stl/inc/xutility

```diff
@@ -541,7 +541,8 @@ struct less_equal<void> {
 template <class _Fx>
 struct _Ref_fn { // pass function object by value as a reference
     template <class... _Args>
-    constexpr decltype(auto) operator()(_Args&&... _Vals) { // forward function call operator
+    constexpr decltype(auto) operator()(_Args&&... _Vals) noexcept(
+        is_nothrow_invocable_v<_Fx&, _Args&&...>) { // forward function call operator

```

Comment:
Errr, this is not supported in C++14... can we use `noexcept(noexcept(_STD invoke(...)))` instead?

Reply:
We can use `_Select_invoke_traits<_Fx&, _Args...>::_Is_nothrow_invocable::value` (`&&` can be omitted as it is added by default).

https://github.com/microsoft/STL/blob/d8ed02d2ec7ba8216848a33dec2b5fb385febf4e/stl/inc/type_traits#L1910